pkgs/development/libraries/getdata/default.nix

```diff
@@ -1,19 +1,15 @@
-{ lib, stdenv, fetchurl, fetchpatch, libtool }:
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, libtool }:
 stdenv.mkDerivation rec {
   pname = "getdata";
-  version = "0.10.0";
-  src = fetchurl {
-    url = "mirror://sourceforge/getdata/${pname}-${version}.tar.xz";
-    sha256 = "18xbb32vygav9x6yz0gdklif4chjskmkgp06rwnjdf9myhia0iym";
+  version = "0.11.0";
+  src = fetchFromGitHub {
+    owner = "ketiltrout";
```

Comment:
Looking at the SourceForge project this seems to be the same owner: https://sourceforge.net/projects/getdata/